submissions/TaniaBondarenko/popup2_task/style.css

```diff
@@ -0,0 +1,101 @@
+/*Common styles*/
+.navigation {
+  background-color: #fc7e08;
+  text-align: end;
+  height: 35px;
+  padding: 5px 5px 15px 5px;
+}
+
+.popup_icon_image {
+  height: 35px;
+  width: 35px;
+}
+.nav_icon {
+  margin-left: 10px;
+  margin-right: 10px;
+}
+
+.nav_icon_image {
+  height: 28px;
+  width: 25px;
+}
+.nav_icon:hover {
+  cursor: pointer;
+  background: rgb(175, 172, 172);
+}
+
+.popup :hover {
+  cursor: pointer;
+  background: rgb(175, 172, 172);
+}
+
+span:hover {
+  cursor: pointer;
+  background: #fc7e08;
+  border: 1px solid #fc7e08;
+}
+
+.li_icon_image {
+  width: 37px;
+  height: 35px;
+}
+
+/*Styles for pressing menu-button*/
+ul {
```

Comment:
It's better to use classes for styling this `ul` element because maybe there are multiple `ul` elements that would be on the page.